### ISSUE TITLE
【SF】【A11y】[SplitButton]: [Missing aria-label for arrow button in splitButton]

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#5967]
### Describe the bug

When used SplitButton, there will be an A11y issue shows missing accessbile text in arrow icon.
![image](https://github.com/SAP/ui5-webcomponents-react/assets/10113427/865e3dfc-e8da-44f2-bd98-83e03c432db1)

### Isolated Example

https://stackblitz.com/edit/github-nvtvkb?file=src%2FApp.tsx

### Reproduction steps

Refer isolated example.

### Expected Behaviour

Add aria-label "more" for arrow button in splitButton

### UI5 Web Components for React Version

1.29.2

### UI5 Web Components Version

1.24.4

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
